CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 7.3.0 - 2023-10-02
+
+- [822](https://github.com/chromaui/chromatic-cli/pull/822) Add `experimental_abortSignal` to Node API to allow canceling builds
+- [819](https://github.com/chromaui/chromatic-cli/pull/819) Migrate from Jest to Vitest, update ESLint config and upgrade Execa
```

Comment:
It looks like these were merged before. I take it they weren't added to the changelog already?

Reply:
I'll clean that up.